esphome/components/sensor/__init__.py

```diff
@@ -188,6 +189,17 @@ def validate_datapoint(value):
     return validate_datapoint({CONF_FROM: cv.float_(a), CONF_TO: cv.float_(b)})
 
 
+_SENSOR_ENTITY_CATEGORIES = {
+    k: v
+    for k, v in cv._ENTITY_CATEGORIES.items()  # pylint: disable=protected-access
```

Comment:
Happy with the overall change, but can we just change this to be non-protected instead?
```suggestion
    for k, v in cv.ENTITY_CATEGORIES.items()
```

Reply:
Yes sure :) I thought that it was protected so everybody uses the function `entity_category` instead